tyk-headless/templates/secrets.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
```

Comment:
Please add the standard set of labels too.

Same in the other charts.

Reply:
Labels are now added.